### DISCUSSION TITLE
Simplex

### DISCUSSION THREAD

**Discussion by valentinajemuovic** [mathnet/mathnet-numerics#841] [Q&A] [answered]
I was looking at the list of topics on this page https://numerics.mathdotnet.com/ and I was not able to find Simplex, so I wanted to ask if it was implemented or if there are any plans to implement it?

**Comment by cdrnet** [comment, marked as answer]
Yes, we do have by now - e.g. NelderMeadSimplex, in the Optimization namespace.
